x509-cert/src/name.rs

```diff
@@ -166,6 +170,25 @@ impl FromStr for RelativeDistinguishedName {
     }
 }
 
+#[cfg(feature = "std")]
+impl TryFrom<Vec<AttributeTypeAndValue>> for RelativeDistinguishedName {
+    type Error = der::Error;
+
+    fn try_from(mut vec: Vec<AttributeTypeAndValue>) -> der::Result<RelativeDistinguishedName> {
+        let slice = vec.as_mut_slice();
+        // This is der_sort from set_of.rs, which is inaccessible from here
+        for i in 0..slice.len() {
+            let mut j = i;
+
+            while j > 0 && slice[j - 1].der_cmp(&slice[j])? == Ordering::Greater {
+                slice.swap(j - 1, j);
+                j -= 1;
+            }
+        }
+        Ok(RelativeDistinguishedName(SetOfVec::try_from(vec)?))
+    }
```

Comment:
```suggestion
    fn try_from(mut vec: Vec<AttributeTypeAndValue>) -> der::Result<RelativeDistinguishedName> {
        Ok(RelativeDistinguishedName(SetOfVec::try_from(vec)?))
    }
```

Reply:
I agree, that's much better. I will move `der_sort()` back.